content/usage/advanced/index.md

```diff
@@ -656,6 +749,8 @@ connected at that time, with UDP streams counting up from port `5600`
 
 {{ simple_pirate_image(src="video-stream-example", width=400, center=true) }}
 
+- H264-encoded streams can be repackaged as WebRTC, and viewed and recorded in
+[Cockpit](https://blueos.cloud/cockpit/docs/latest/usage/advanced/#video-widgets)
```

Comment:
We might want to mention that it works for H265 too, but only for Cockpit on macOS (for now)?